app/jenkins-for-jira-ui/src/components/MainPage/MainPage.styles.tsx

```diff
@@ -0,0 +1,41 @@
+import { css } from '@emotion/css';
+
+export const mainPageContainer = css`
+	margin: 0 auto;
```

Comment:
Chatted with Dom and confirmed we want to fix the width to 936px and centre-align.